src/core/Akka.Cluster/Cluster.cs

```diff
@@ -81,6 +81,11 @@ internal static bool IsAssertInvariantsEnabled
         /// </summary>
         public UniqueAddress SelfUniqueAddress { get; }
 
+        /// <summary>
+        /// Used to retain the <see cref="InfoLogger"/> instance that decorates the cluster.
+        /// </summary>
+        internal InfoLogger CurrentInfoLogger { get; }
```

Comment:
I don't have a nice clean `object` abstraction I can use to hide this stuff the same way we do on the JVM, and `static` members would have some unintended consequences here. Decided to expose the `InfoLogger` as an `internal` field, but I mostly just ended up using the `LogInfo` methods that were already built into the `Cluster` class instead.